backend/src/main/java/planit/people/preparation/GoogleConnector/GoogleConnector.java

```diff
@@ -112,17 +110,13 @@ public Oauth2 oauth2Service() {
 
     private void setRefreshAndExpiry() {
         try {
-            InputStream in = new FileInputStream(CREDENTIALS_FILE_PATH);
-            GoogleClientSecrets clientSecrets = GoogleClientSecrets.load(JSON_FACTORY, new InputStreamReader(in));
-            String clientId = clientSecrets.getDetails().getClientId();
-            String clientSecret = clientSecrets.getDetails().getClientSecret();
             GoogleTokenResponse response = new GoogleAuthorizationCodeTokenRequest(
                     new NetHttpTransport(), GsonFactory.getDefaultInstance(),
-                    clientId, clientSecret,
+                    CLIENT_ID, CLIENT_SECRET,
                     this.code, "http://localhost:3000")
```

Comment:
Can we change it to https://planit-custom-domain.loca.lt/`? It's required for the signin in Docker and it should also work locally (I checked before)